src/cancel-reminders/lib/db.test.ts

```diff
@@ -33,16 +50,15 @@ describe('cancelPendingReminders', () => {
 			identity_id: identityId,
 			reminder_period: '2021-02-01',
 		});
```

Comment:
We can probably drop the explicit `identityId` here as we don't need to reference it again for cancelling - so just:

```js
		const r1 = createOneOffReminder({
			reminder_period: '2021-01-01',
		});
		const r2 = createOneOffReminder({
			reminder_period: '2021-02-01',
		});
```

Reply:
(I think we can do this in most of the tests)

Reply:
👍